src/locale/locales/fr/messages.po

```diff
@@ -3111,6 +3127,10 @@ msgstr "Rejoignez Bluesky"
 msgid "Join the conversation"
 msgstr "Participez à la conversation"
 
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:493
+msgid "Joined {0}"
+msgstr "Inscrit le {0}"
```

Comment:
Even though we could say it's _technically_ the `compte` that was `inscrit`, it wouldn't work here. Also, adjective are very rarely _épicène_, so I am afraid that the _point médian_ is the only way I can find (especially in my bed at 1am). 

```suggestion
msgstr "Inscrit·e le {0}"
```

Reply:
Oops, completely forgot about _la langue épicène_ in this context 😬